WORKSPACE

```diff
@@ -22,7 +22,12 @@ git_repository(
 
 bind(
     name = "protobuf_clib",
-    actual = "@tf//google/protobuf:protobuf",
+    actual = "@tf//google/protobuf:protoc_lib",
+)
+
+bind(
+    name = "protobuf_compiler",
```

Comment:
Since we're going to use grpc defined in Tensorflow's workspace, we can remove protobuf_compiler and protobuf_clib entirely from here right?